packages/communication-react/CHANGELOG.md

```diff
@@ -4,21 +4,20 @@ This log was last generated on Fri, 21 Oct 2022 23:01:52 GMT and should not be m
 
 <!-- Start content -->
 
-## [1.4.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.4.0)
+## [1.4.0](https://github.com/azure/communication-ui-library/tree/1.4.0)
 
-Fri, 21 Oct 2022 23:01:52 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.3.2-beta.1...@azure/communication-react_v1.4.0)
+Fri, 21 Oct 2022 23:01:52 GMT
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.3.1...1.4.0)
+
+This stable release contains a slew of bug fixes, some of them for bugs filed by our users in our GitHub repository. Thanks!
+The release also contains some small API additions to support these fixes.
 
 ### Minor changes
 
 - `@internal/react-composites`
-  - Call adapter can join a room ([PR #2063](https://github.com/azure/communication-ui-library/pull/2063) by miguelgamis@microsoft.com)
   - Add callEndReason to the onCallEnded event and fix the event to trigger before the composite page transition. ([PR #2201](https://github.com/azure/communication-ui-library/pull/2201) by 2684369+JamesBurnside@users.noreply.github.com)
```

Comment:
`callEndReason` was subsequently removed (while we wait for calling to add a better api for this), but the onCallEnded event was still fixed to be called at the correct time

Reply:
So this is a patch change then?

Reply:
Yeah

Reply:
Looks like it still needed some API changes for the event so leaving it here.

but I realized that some changes in patches below also had API changes, so moving them up here.